src/styles/styles.js

```diff
@@ -37,24 +37,15 @@ const styles = {
         marginLeft: 8,
     },
 
+    mt2: {
```

Comment:
Please don't change the indentation utils here. If you need `marginTop: 20`, use `mt5`

Reply:
Also please make sure that these are arranged sequentially like `mt1, mt2, mt3...`